# [\#3059 Issue](https://github.com/ethereum/solidity/issues/3059) `closed`: Do not abort compilation on first unknown identifier

#### <img src="https://avatars.githubusercontent.com/u/138426?u=3117125771b06e3aa8da468c8f41e4038d717974&v=4" width="50">[federicobond](https://github.com/federicobond) opened issue at [2017-10-09 23:45](https://github.com/ethereum/solidity/issues/3059):

Since the reference/name resolving compiler pass is independent of other passes, we can probably continue resolving names after the first error.

#### <img src="https://avatars.githubusercontent.com/u/9073706?v=4" width="50">[chriseth](https://github.com/chriseth) commented at [2018-02-13 15:53](https://github.com/ethereum/solidity/issues/3059#issuecomment-365309362):

Can this be closed now?

#### <img src="https://avatars.githubusercontent.com/u/20340?v=4" width="50">[axic](https://github.com/axic) commented at [2018-02-13 16:15](https://github.com/ethereum/solidity/issues/3059#issuecomment-365316770):

Fixed by #3360.


-------------------------------------------------------------------------------



[Export of Github issue for [ethereum/solidity](https://github.com/ethereum/solidity). Generated on 2024.12.15 at 06:45:24.]
